frontend/src/routes/Governance/discovered/ByCluster/CertByCluster.tsx

```diff
@@ -0,0 +1,38 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { PageSection } from '@patternfly/react-core'
+import { DiscoverdPolicyItem } from '../useFetchPolicies'
+import { AcmEmptyState, AcmTable } from '../../../../ui-components'
+import { useTranslation } from '../../../../lib/acm-i18next'
+import { useRecoilValue, useSharedAtoms } from '../../../../shared-recoil'
+import { Box } from '@mui/material'
+import { ByClusterCols, DiscoveredViolationsCard, policyViolationSummary } from './common'
+
+export default function CertByCluster({ policies }: Readonly<{ policies: DiscoverdPolicyItem[] }>) {
+  const { t } = useTranslation()
+  const { channelsState, helmReleaseState, subscriptionsState } = useSharedAtoms()
+  const helmReleases = useRecoilValue(helmReleaseState)
+  const subscriptions = useRecoilValue(subscriptionsState)
+  const channels = useRecoilValue(channelsState)
+
+  return (
+    <PageSection>
+      <Box mb={3} maxWidth={500}>
+        <DiscoveredViolationsCard
+          policyKind="CertificatePolicy"
+          policyViolationSummary={policyViolationSummary(policies ?? [])}
+        />
+      </Box>
+      <AcmTable<DiscoverdPolicyItem>
+        id="certificatePolicyByCluster"
+        columns={ByClusterCols(t, helmReleases, subscriptions, channels)}
+        keyFn={(item) => item.cluster}
+        items={policies ?? []}
+        emptyState={
+          <AcmEmptyState title={t(`No Certificate policy found`)} message={t('There are no Certificate policies')} />
```

Comment:
To be confirmed by @dockerymick 
```suggestion
          <AcmEmptyState title={t(`No certificate policies were found`)} message={t('The search results returned no certificate policies')} />
```

Reply:
I recommend the following for the first statement, "You don't have any certificate policies"
I agree with the second statement